content/applications/hr/employees/new_employee.rst

```diff
@@ -0,0 +1,269 @@
+==================
+Add a new employee
+==================
+
+When a new employee is hired, the first step is to create a new employee form. Starting in the
+:guilabel:`Employees` app default view, click the :guilabel:`Create` button to access a new employee
+form. Fill out the required information (underlined in bold) and any additional details, then click
+:guilabel:`Save`.
+
+.. image:: new_employee/employee-new.png
+   :align: center
+   :alt: Create a new employee card.
+
+.. note::
+   The current company phone number and name is populated in the :guilabel:`Work Phone` and
+   :guilabel:`Company` fields.
+
+General information
+===================
+
+Required fields
+---------------
+
+- :guilabel:`Name`: Enter the employee's name.
+- :guilabel:`Company`: Select the company from the drop-down menu that the new employee is hired by,
+  or create a new company by typing the name in the field.
+- :guilabel:`Working Hours`: In the :guilabel:`Work Information` tab, select the desired
+  :guilabel:`Working Hours` from the drop-down menu.
+
+.. image:: new_employee/working-hours.png
+   :align: center
+   :alt: Working Hours are located in the Work Information tab.
+
+.. note::
+   :guilabel:`Working Hours` are related to a company's working times, and an employee cannot have
+   working hours that are outside of a company's working times.
+
+   Each individual working time is company-specific, so for multi-company databases, each company
+   needs to have it's own working hours set.
```

Comment:
I believe this should be 'its' (possessive pronoun) instead of 'it’s' (short for 'it is')
```suggestion
   needs to have its own working hours set.
```